app/pods/cluster/template.hbs

```diff
@@ -37,39 +37,24 @@
   {{/dashboard-module}}
 
   {{#dashboard-module label='Nodes'}}
-    {{#if model.nodes.length}}
-      <table class="table">
-        <thead>
-        <tr>
-          <th>Ping</th>
-          <th>Stats</th>
-          <th>Node</th>
-        </tr>
-        </thead>
-        <tbody>
-        {{#each model.nodes as |node|}}
-          <tr>
-            <td>
-              {{#link-to 'riak-ping' (query-params nodeId=node.id clusterId=model.clusterId)
-              classNames='btn btn-sm btn-primary' }}
-                Node Ping
-              {{/link-to}}
-            </td>
-            <td>
-              {{#link-to 'node-stats' (query-params nodeId=node.id clusterId=model.clusterId)
-              classNames='btn btn-sm btn-primary' }}
-                Node Stats
-              {{/link-to}}
-            </td>
-            <td>
-              {{node.id}}
-            </td>
-          </tr>
-        {{/each}}
-        </tbody>
-      </table>
+    {{#if model.riakNodes}}
+      <ul class="button-list">
+          {{#each model.riakNodes as |node|}}
+              <li>
+                  {{#link-to 'node-stats'
+                  (query-params nodeId=node.id clusterId=model.id)
+                  classNameBindings="!node.available:danger"}}
+                      {{node.id}}
+                      <span {{bind-attr class=":glyphicon node.available:glyphicon-ok-circle:glyphicon-ban-circle"}}
```

Comment:
This gives a deprecation error:

```
DEPRECATION: The `bind-attr` helper ('ember-riak-explorer/pods/cluster/template.hbs' @ L48:C30) is deprecated in favor of HTMLBars-style bound attributes
```

(They're removing `bind-attr` in 2.0, looks like. If you don't mind, can you refactor this to use the `<div class={{if isActive "active" "disabled"}}></div>` syntax?)

Also, do you have jsHint / compile warnings suppressed on your end?